libs/txnbuild/mint_credit.go

```diff
@@ -38,6 +40,7 @@ func (mintCredit *MintCredit) BuildXDR() (vxdr.VeloOp, error) {
 	return vxdr.VeloOp{Body: body}, nil
 }
 
+// FromXDR for MintCredit initialises the vtxnbuild struct from the corresponding xdr Operation.
```

Comment:
FromXDR for MintCredit initialises the vtxnbuild struct from the corresponding **XDR** Operation.